packages/formation-react/src/components/ErrorableCheckbox/ErrorableCheckbox.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ErrorableCheckbox from './ErrorableCheckbox';
+
+export default {
+  title: 'Library/ErrorableCheckbox',
+  component: ErrorableCheckbox,
+};
+
+const Template = (args) => <ErrorableCheckbox {...args} />;
+
+const defaultArgs = {
+  required: true,
+  errorMessage: 'Error message',
```

Comment:
What do you think about making the default without an error message and then making another story with the error state?

Reply:
ahh yea good idea the default should be the primary state ill update it tks